### ISSUE TITLE
new feature proposal: Glob match used in multiple entry.

### ISSUE THREAD

**Issue by HawtinZeng** [emosheeep/vite-plugin-virtual-mpa#64]
Refer to this link: https://github.com/vitejs/vite/issues/3429, I think it's neccessary to offer glob match.

**Comment by emosheeep** [comment]
you can use `fast-glob` to customize your own logic.

**Comment by HawtinZeng** [comment]
Yes, but I still think it's better to leave config file clean...

**Comment by emosheeep** [comment]
Yep, but I have to work,  I'm glad to handle a PR. Otherwise it will took a long time util I have a break in someday.

**Comment by HawtinZeng** [comment]
I'm glad to offer a PR later.
